.github/workflows/npm_publish.yml

```diff
@@ -24,9 +25,12 @@ jobs:
           node-version: 12
       - run: npm ci
       - run: npm run test
-      - uses: paambaati/codeclimate-action@v2.2.6
+        env:
+          REDIS_PORT: ${{ job.services.redis.ports[6379] }}
```

Comment:
do we need to define the `REDIS_PORT` env twice or can we remove one of these?

Reply:
we need both beacuse the code climate action also need it to run the tests